src/lib/style/components/_entity-metadata.scss

```diff
@@ -0,0 +1,72 @@
+@use '../abstracts/variables' as SRC;
+
+.EntityMetadata {
+  .Tabs {
```

Comment:
I think this is a core style that we should share (for example, I should change the Download List v2 UI to use these instead of rolling my own).